src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,55 @@
+body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: ($cell-size * $columns-number) + 6px;
+  padding-inline: 10px;
+
+  &__day {
+    height: $cell-size;
+    width: $cell-size;
+    background-color: $cell-color;
+    box-sizing: border-box;
+    border: 1px solid black;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    font-family: $main-font;
+    font-size: $text-size;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $cell-hover-color;
+      transform: translateY(-20px);
+      transition-duration: 500ms;
+    }
+  }
+
+  @for $i from 1 through 31 {
+    & :nth-child(#{$i})::before {
+      content: '#{$i}';
+      display: block;
```

Comment:
consider moving styles to a regular class instead of generating the same styling for each of the 31 elements
```suggestion

```